actors/miroslav-nemec.json

```diff
@@ -0,0 +1,9 @@
+
+
+{
+    "name": "Miroslav Nemec",
+    "birthname": "Miroslav Nemec",
+    "birthdate": "1954-06-26",
+    "birthplace": "Zagreb, Croatia"
```

Comment:
Birth place should be: `Zagreb, Croatia, Yugoslavia`

Reply:
Fixed